venues-on-map/demo.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Boilerplate map initialization code starts below:
+ */
+// Step 1: initialize communication with the platform
+// In your own code, replace variable window.apikey with your own apikey
+var platform = new H.service.Platform({
+  apikey: window.apikey
+});
+var defaultLayers = platform.createDefaultLayers();
+
+// Step 2: initialize a map
+var map = new H.Map(document.getElementById('map'), defaultLayers.vector.normal.map, {
+  zoom: 18,
+  center: { lat: 47.452353, lng: 8.562455 },
+  pixelRatio: window.devicePixelRatio || 1
+});
+
+// add a resize listener to make sure that the map occupies the whole container
+window.addEventListener('resize', () => map.getViewPort().resize());
+
+// Step 3: make the map interactive
+// MapEvents enables the event system
+// Behavior implements default interactions for pan/zoom (also on mobile touch environments)
+var behavior = new H.mapevents.Behavior(new H.mapevents.MapEvents(map));
+
+// Step 4: create the default UI component, for displaying bubbles
+var ui = H.ui.UI.createDefault(map, defaultLayers);
+
+// Step 5: load venue data
+// Get instance of venues service using valid apikey for venues
+const venuesService = platform.getVenuesService({ apikey: 'QICW7garcjxE7C7sSguJcNolMZXqYCJ9m5o6Qq3ygjg' });
+
+// Venues provider interacts with tile layer to visualize and control the venue map
+const venuesProvider = new H.venues.Provider();
+
+// Venues service provides a loadVenue method
+venuesService.loadVenue(7348).then((venue) => {
```

Comment:
Would be nice to wrap venues specific logic in the function and push it above the standard boilerplate code in line with the other examples. It makes it easier to spot the code that changes from one example to the other.